CON-1742 KnowBe4 App Doc (#1955)
SumoLogic/sumologic-documentation: docs/integrations/saas-cloud/knowbe4.md

---
id: knowbe4
title: Sumo Logic App for KnowBe4
sidebar_label: KnowBe4

> description: The Sumo Logic App for KnowBe4 offers functionality for monitoring and analyzing your organization's KnowBe4 Phishing Security logs.

Reviewer: ```suggestion
description: The Sumo Logic App for KnowBe4 offers functionality for monitoring and analyzing KnowBe4 Phishing Security logs.
```

Reviewer: Your org doesn't add much value here.

Author: Agree. Updated the code.

---

import useBaseUrl from '@docusaurus/useBaseUrl';

> <img src={useBaseUrl('img/send-data/knowbe4.png')} alt="icon" width="50" />

Author: Updated the Code

> The KnowBe4 App for Sumo Logic allows you to easily integrate data from KnowBe4, the world's largest security awareness training and simulated phishing platform, into your Sumo Logic account. This App provides an overview of your organization's phishing security testing performance. It includes metrics on the total number of tests, active tests, recipient users, failed recipient users, and distribution of status. The distribution of Phishing Security Tests by difficulty and long-running Phishing Security Tests panels offers insights into the difficulty level of the phishing tests and identifies tests that have been running for an extended period. The Phish Prone Percentage panel helps you understand the percentage of users who are at higher risk of falling for a phishing attack. The Geo Location of Failed Users panel provides geographic insights into where failed users are located. The Phishing Security Tests Summary and Phish Failures Summary panels provide high-level summaries of the organization's testing performance. Finally, the Top Failed Users panel identifies users who have failed multiple tests and may require additional training.

Reviewer 2: ```suggestion
The KnowBe4 App for Sumo Logic allows you to easily integrate data from KnowBe4, a security awareness training and simulated phishing platform, into your Sumo Logic account. This App provides an overview of your organization's phishing security testing performance. It includes metrics on the total number of tests, active tests, recipient users, failed recipient users, and distribution of status. 

The distribution of **Phishing Security Tests** by difficulty and long-running **Phishing Security Tests** panels offer insights into the difficulty level of the phishing tests and identify tests that have been running for an extended period. The app's **Phish Prone Percentage** dashboard panel helps you understand the percentage of users who are at higher risk of falling for a phishing attack. 

The **Geo Location of Failed Users** dashboard panel provides geographic insights into where failed users are located. The **Phishing Security Tests Summary** and **Phish Failures Summary** dashboard panels provide high-level summaries of the organization's testing performance. Finally, the **Top Failed Users** panel identifies users who have failed multiple tests and may require additional training.
```

Author: Updated the Code

## Log Types

> The Sumo Logic App for KnowBe4 consumes Phishing Security logs, refer to the [Phishing Security Tests](https://developer.knowbe4.com/rest/reporting#tag/Phishing/paths/~1v1~1phishing~1security_tests/get) and [Recipient Results](https://developer.knowbe4.com/rest/reporting#tag/Phishing/paths/~1v1~1phishing~1security_tests~1{pst_id}~1recipients/get) documentation.

Reviewer 2: ```suggestion
The Sumo Logic App for KnowBe4 consumes Phishing Security logs. Refer to the [KnowBe4 Phishing Security Tests](https://developer.knowbe4.com/rest/reporting#tag/Phishing/paths/~1v1~1phishing~1security_tests/get) and [Recipient Results](https://developer.knowbe4.com/rest/reporting#tag/Phishing/paths/~1v1~1phishing~1security_tests~1{pst_id}~1recipients/get) documentation.
```

Author: Updated the Code

email, scheduled_at, delivered_at, opened_at, clicked_at, replied_at, attachment_opened_at, macro_enabled_at, data_entered_at, qr_code_scanned, reported_at, bounced_at, ip, ip_location, browser, browser_version, os nodrop
| where ip_location matches "{{user_location}}"
| count_distinct(recipient_id)
```

## Installing the KnowBe4 App

Locate and install the app from the **App Catalog**. To see a preview of the dashboards included with the app before installing, click **Preview Dashboards**.
Before you begin, collect logs from KnowBe4 and ingest them into Sumo Logic. Refer to the [KnowBe4 Cloud-to-Cloud Integration](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/knowbe4-api-source/) to create the source and use the same source category while installing the app.

To install the app, follow the steps below:
1. From the **App Catalog**, search for the app and select it.
1. Select **Add Integration** button to install the app.

> 1. Configure **KnowBe4 App** using the steps described in the [KnowBe4 Cloud-to-Cloud Source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/knowbe4-api-source/). If you already have set up your data, skip this step by clicking on **Next**.

Author: Updated the code.

1. Complete the following fields:
   1. **Data Source**. Select either of these options for the data source:
      * Choose **Source Category** and then choose a source category from the list.
      * Select **Enter a Custom Data Filter** and type in a custom source category that starts with an underscore. For example, `_sourceCategory=MyCategory`.
    2. **Folder Name**. You can retain the existing name, or enter a name of your choice for the app. 
    3. Select the **Location in Library** (the default is the **Personal** folder in the library), or click **New Folder** to add a new folder.
1. Click **Next**.

Once an app is installed, it will appear in your **Personal** folder, or other folder that you specified. You can share it with your organization.

> The panels will begin to fill automatically. It's worth noting that each panel gradually fills with data that matches the time range query and has been received since the panel was created. The results will not be available right away, but with some patience, you will be able to view full graphs and maps.

Reviewer: ```suggestion
The panels will begin to fill automatically. It's worth noting that each panel gradually fills with data that matches the time range query and has been received since the panel was created. The results will not be available right away, but you will be able to view full graphs and maps.
```

Author: Updated the code.

## Viewing KnowBe4 Dashboards​

> * All dashboards have a set of filters that you can apply to the entire dashboard, as shown in the following example. Click the funnel icon in the top dashboard menu bar to display a scrollable list of filters that are applied across the entire dashboard.
> 
>  You can use filters to drill down and examine the data on a granular level.
> 
> * Each panel has a set of filters that are applied to the results for that panel only, as shown in the following example. Click the funnel icon in the top panel menu bar to display a list of panel-specific filters.

Reviewer 2: ```suggestion
All dashboards have a set of filters that you can apply to the entire dashboard, as shown in the following example. Click the funnel icon in the top dashboard menu bar to display a scrollable list of filters that are applied across the entire dashboard.

You can use filters to drill down and examine the data on a granular level.

Each panel has a set of filters that are applied to the results for that panel only, as shown in the following example. Click the funnel icon in the top panel menu bar to display a list of panel-specific filters.
```

Author: Updated the Code
